RELEASING/release-notes-4-0/README.md

```diff
@@ -0,0 +1,145 @@
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release Notes for Superset 4.0.0
+
+4.0.0 brings a plethora of exciting changes to Superset. We have introduced several breaking changes to improve the overall architecture and scalability of our codebase. These changes may require some code updates, but they are designed to enhance performance and maintainability in the long run. We have also upgraded various dependencies to their latest versions and deprecated certain features that are no longer aligned with our long-term roadmap. We encourage all developers to carefully review the `CHANGELOG.md` and `UPDATING.md` files and update their code accordingly. Even though, our main focus was on code cleanup, this release still contain exciting new features and mark a significant milestone for us, paving the way for our continued growth and success.
+
+Here are some of the highlights of this release.
+
+### Alerts and Reports modal redesign
+
+The Alerts and Reports modal has been [redesigned](https://github.com/apache/superset/discussions/25729) to improve the user experience and make it more intuitive. The new design is has the following goals:
+
+- Declutter the interface by providing a cleaner, more organized layout
+- Create a linear setup process with the necessary options in a step-by-step manner to make alert/report setup more intuitive
+- Prepare the interface for additional features that will be introduced in future releases
+
+<div>
+    <img src="media/alert-modal-1.png" alt="Image" width="33%">
+    <img src="media/alert-modal-2.png" alt="Image" width="33%">
+    <img src="media/alert-modal-3.png" alt="Image" width="33%">
+</div>
+
+### Tags
+
+Tags are available using the `TAGGING_SYSTEM` feature flag. They address many of the [requests made by the community](https://github.com/apache/superset/discussions/19194) and aim to make it easier to organize and curate charts, dashboards, and saved queries, allowing for effortless data discovery and collaboration within an organization. Users can create flexible and customizable tags for each piece of content, enabling different ways of organizing assets. Tags are also available in the API, allowing for programmatic access to tag-related operations.
+
+<div>
+    <img src="media/tags-1.png" alt="Image" width="100%">
+    <img src="media/tags-2.png" alt="Image" width="100%">
+</div>
+
+### New CHANGELOG format
+
+We changed the structure of the `CHANGELOG.md` file in [#26800](https://github.com/apache/superset/pull/26800) to better organize the contents of each release and also to deal with GitHub size limitations when displaying the file. Now every release will have its own file at `CHANGELOG/<version>.md`. The main `CHANGELOG.md` file is now an index with links to all releases.
+
+### Improved drag and drop experience when editing a dashboard
```

Comment:
How about highlighting the explore D&D experiences together? (#27558)

Reply:
@justinpark It would be awesome to include something about this as I think it's a great improvement. We just need to make sure the PR is merged and included in the branch first.

Reply:
Thank you!